tests/integration/test_integration.py

```diff
@@ -22,30 +22,13 @@
 
 
 class TestVaultK8s:
-    @pytest.fixture(scope="module")
-    @pytest.mark.abort_on_fail
-    async def charm(self, ops_test):
-        ops_test.destructive_mode = False
-        charm = await ops_test.build_charm(".")
-        return charm
-
-    @pytest.fixture()
-    async def cleanup(self, ops_test):
-        try:
-            await ops_test.model.remove_application(
-                app_name=APPLICATION_NAME, block_until_done=True
-            )
-        except JujuError:
-            pass
-
     @staticmethod
     async def wait_for_load_balancer_address(kubernetes: Kubernetes, timeout: float = 60):
```

Comment:
We should also add return typing

Reply:
Added. I've also added additional docstrings and fixed a return type.